src/stratis_cli/_actions/_report_pool.py

```diff
@@ -154,7 +155,8 @@ def selection_func(_uuid, info):
         if stopped_pool is None:
             raise StratisCliResourceNotFoundError("list", selection_value)
 
-        # Substitute lsblk call here
+        # Ignore bandit B603 errors.  Input comes from D-Bus and has
+        # been processed.
```

Comment:
Try to get a single list of devnodes, just as you did with the started pools. You should be able to use roughly the same style of list comprehension as you did there, and eliminate the ```for-loop``` entirely.

Reply:
Resolved.